sdk/cosmos/azure-cosmos/dev_requirements.txt

```diff
@@ -1,3 +1,4 @@
+aiohttp>=3.0
```

Comment:
Are these dev requirements only used for dev purposes or does this file gets shipped?

Reply:
these dev_requirements are what we tell our users to install in order to use our SDK - and is also used by the pipelines to install the packages they need to run the tests. In this case we had to add aiohttp since otherwise the async tests won't run.